test/meson.build

```diff
@@ -49,6 +52,9 @@ foreach test_name : tests
     link_with : librauc,
     c_args : '-DTEST_SERVICES="' + meson.build_root() + '"',
     include_directories : incdir,
+    install_mode : 'rwxr-xr-x',
+    install : get_option('install-tests'),
+    install_dir : get_option('libdir') / 'rauc' / 'tests',
```

Comment:
The test installation directory should be defined as a variable before using it. Either in meson.build or test/meson.build.